### PR TITLE
Lr22

### PR DESCRIPTION
**Pull request by archick12**

### CODE REVIEW

File: src/main/java/framework/RestConfiguration.java [hunk 1]
```diff
@@ -9,6 +9,6 @@
     public void configure(){
         RestAssured.baseURI= "http://jira.hillel.it";
         RestAssured.port =8080;
-        //RestAssured.basePath="/people";
+        //RestAssured.basePath="";
```
Comment (archick12) [inline src/main/java/framework/RestConfiguration.java:12]: Why do you need this?

File: src/test/java/apiTests/TryApi.java [hunk 2]
```diff
@@ -2,41 +2,52 @@
...
     @BeforeSuite
     public void setupMethod(){
         RestAssured.baseURI = "http://jira.hillel.it";
         RestAssured.port = 8080;
-        JSONObject login = new JSONObject();
-        login.put("username",username);
-        login.put("password",password);
+
+        login = new Login();
+        login.setLogin("webinar5");
```
Comment (archick12) [inline src/test/java/apiTests/TryApi.java]: Can this setters be replaced with constructor?
 new Login("webinar5", "webinar5");

File: src/test/java/apiTests/TryApi.java [hunk 3]
```diff
@@ -2,41 +2,52 @@
...
 
 public class TryApi {
-    String username = "webinar5";
-    String password = "webinar5";
     String sessionId;
     String issueId;
     String commentId;
+    String set;
+
+    public static Login login;
```
Comment (archick12) [inline src/test/java/apiTests/TryApi.java]: I would recommend to rename Login to Authorization

File: src/main/java/framework/RestConfiguration.java (hunk 1 above)
Comment (Artem100) [reply src/main/java/framework/RestConfiguration.java:12]: I tried to take out this class from the tests separately, as annotation
@BeforeTest

**Pull request merged**